.travis.yml

```diff
@@ -11,44 +11,27 @@ addons:
 
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27-django-111
-
-    - python: 3.4
-      env: TOXENV=py34-django-111
-    - python: 3.4
-      env: TOXENV=py34-django-20
-
-    - python: 3.5
-      env: TOXENV=py35-django-111
```

Comment:
Strictly speaking django 1.11 could be supported, but since it is quite old and extended support has ended, I think it is fair to drop it in a major release. I think the major reason for sticking with 1.11 is Python 2, which I want to drop anyways as removing this makes life a lot easier.

Reply:
Fair enough. I'm aware of a few Django apps running with 1.11 and Python 3, but support had indeed been dropped. I'm curious to see what the Django folks do if there is a security bug in 1.11, but that's another story.